morphocut/image.py

```diff
@@ -134,7 +152,13 @@ def transform_stream(self, stream):
 @ReturnOutputs
 @Output("extracted_image")
 class ExtractROI(Node):
-    """Return the extracted region/image
+    """
+    Return the extracted region/image
+
+    Args:
+        image (RawOrVariable or Variable[RawOrVariable]): An image file from which regions are to be extracted.
```

Comment:
image (np.ndarray or Variable[np.ndarray]): Image from which regions are to be extracted.